### PR TITLE
Feat/logging playground

### PR DESCRIPTION
**Pull request by renesweet24**

### CODE REVIEW

**Comment by pamanseau** [comment]
```╭───────────────────── Traceback (most recent call last) ──────────────────────╮
│ /root/blockchain-data-subnet/neurons/miners/bitcoin/funds_flow/indexer.py:19 │
│ 3 in <module>                                                                │
│                                                                              │
│   190 │   end_height_str = os.getenv('BITCOIN_INDEXER_END_BLOCK_HEIGHT', '-1 │
│   191 │   in_reverse_order_str = os.getenv('BITCOIN_INDEXER_IN_REVERSE_ORDER │
│   192 │                                                                      │
│ ❱ 193 │   logger.info("BITCOIN_INDEXER_IN_REVERSE_ORDER", in_reverse_order = │
│   194 │                                                                      │
│   195 │   if start_height_str is None:                                       │
│   196 │   │   logger.info("Please specify BITCOIN_INDEXER_START_BLOCK_HEIGHT │
│                                                                              │
│ /usr/lib/python3.9/logging/__init__.py:1442 in info                          │
│                                                                              │
│   1439 │   │   logger.info("Houston, we have a %s", "interesting problem", e │
│   1440 │   │   """                                                           │
│   1441 │   │   if self.isEnabledFor(INFO):                                   │
│ ❱ 1442 │   │   │   self._log(INFO, msg, args, **kwargs)                      │
│   1443 │                                                                     │
│   1444 │   def warning(self, msg, *args, **kwargs):                          │
│   1445 │   │   """                                                           │
╰──────────────────────────────────────────────────────────────────────────────╯
TypeError: _log() got an unexpected keyword argument 'in_reverse_order'```

File: template/base/neuron.py [hunk 1]
```diff
@@ -107,6 +129,7 @@ def __init__(self, config=None):
         mandatory_config['uid'] = self.uid
         mandatory_config['ip'] = re.search(r"/ipv4/([^:]*)", self.metagraph.addresses[self.uid]).group(1)
         mandatory_config['hotkey'] = self.wallet.hotkey.ss58_address
+        mandatory_config['coldkey'] = self.metagraph.coldkeys[self.uid]
```
Comment (aph5nt) [inline template/base/neuron.py:132]: this will not work, this will prompt for password and hold execution

File: neurons/remote_config.py [hunk 2]
```diff
@@ -65,7 +65,7 @@ def load_remote_config(self):
                     break  # Break the loop if successful
                 except requests.exceptions.RequestException as e:
                     retries += 1
-                    bt.logging.error("Attempt failed to update config from", retries = retries, config_url = self.config_url, error = {'exception_type': e.__class__.__name__,'exception_message': str(e),'exception_args': e.args})
+                    bt.logging.error("Attempt failed to update config", retries = retries, config_url = self.config_url, error = {'exception_type': e.__class__.__name__,'exception_message': str(e),'exception_args': e.args})
```
Comment (aph5nt) [inline neurons/remote_config.py:68]: for some reason if reties as int is thorws serialization error, my work around was to str(retires),m pls double check

File: template/base/neuron.py (hunk 1 above)
Comment (renesweet24) [reply template/base/neuron.py:132]: It doesn't require password. It has been already tested.

**Pull request merged**